webpack.config.js

```diff
@@ -91,6 +91,12 @@ module.exports = {
       'process.env.FIREBASE_DATABASE_URL': JSON.stringify(
         process.env.FIREBASE_DATABASE_URL
       ),
+      'process.env.DEVELOPER_KEY': JSON.stringify(
+        process.env.DEVELOPER_KEY
+      ),
+      'process.env.CLIENT_ID': JSON.stringify(
+        process.env.CLIENT_ID
+      ),
```

Comment:
perhaps prefix these with `GOOGLE`? Also can you add them to `.env.example`, and likewise some documentation to the `README.md` file?